unc_classification_tab_data/models/bootstrapped_classifier.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import copy
+
+
+class BootstrappedClassifier:
+    """Wrapper class for an ensemble of classifiers, trained on different bootstraps of the data.
+
+    Parameters
+    ----------
+    classifier:
+        A classifier that has the functions 'fit' and 'predict_proba'
+    n: int
+        The ensemble size
+    bootstrap_size: int
+        The size of the bootstrapped samples.
+    """
+
+    def __init__(self, classifier, n: int, bootstrap_size: int):
+        self.classifiers = [copy.copy(classifier) for i in range(n)]
+        self.n_bootstraps = n
+        self.bootstrap_size = bootstrap_size
+
+    def fit(self, X: np.ndarray, y: np.ndarray):
```

Comment:
Is this not going to work at all with pandas? (i.e. do we **have** to convert to numpy beforehand?)

Reply:
Yes, for this experiment code, I thought it might make things unnecessarily complicated. But I can also change it.

Reply:
No that's fine :)